packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -14,12 +18,64 @@ const releaseController = {
     await permissionsManager.validateQuery(ctx.query);
     const query = await permissionsManager.sanitizeQuery(ctx.query);
 
-    ctx.body = await getService('release', { strapi }).findMany(query);
+    const { results, pagination } = await getService('release', { strapi }).findMany(query);
+
+    // Format the data object
+    const data = results.map((release: ReleaseWithPopulatedActions) => {
+      // TODO: Not sure how else to type this otherwise the actions is always unknown
+      const { actions, ...releaseData } = release;
+
+      return {
+        ...releaseData,
+        actions: {
+          meta: {
+            count: actions.count,
+          },
+        },
+      };
+    });
+
+    ctx.body = { data, pagination };
+  },
+
+  async findOne(ctx: Koa.Context) {
+    const id: Release['id'] = ctx.params.id;
+
+    const permissionsManager = strapi.admin.services.permission.createPermissionsManager({
+      ability: ctx.state.userAbility,
+      model: RELEASE_MODEL_UID,
+    });
+
+    await permissionsManager.validateQuery(ctx.query);
+    const query = await permissionsManager.sanitizeQuery(ctx.query);
```

Comment:
Not sure about this, for this first iteration we are not filtering the actions so I don't see what we can pass as query 🤔 And I'm not sure how we gonna do it in the future

Reply:
yeah you are right I removed the query